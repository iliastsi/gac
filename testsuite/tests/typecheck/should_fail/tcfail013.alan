redefError() : proc
  foo : int;
  foo : byte;
{
  foo = 5;
  foo = 'a';
}
