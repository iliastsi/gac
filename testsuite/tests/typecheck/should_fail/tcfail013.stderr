tcfail013.alan:3:3: Error: Conflicting definitions for `foo'
	Bound at: tcfail013.alan:2:3-5
	          tcfail013.alan:3:3-5

tcfail013.alan:5:3: Error: Type mismatch at `foo = 5;'
	Lvalue is of type `byte' but Rvalue is of type `int'

